### ISSUE TITLE
It doesnt let you open drum sniffer and guitar sniffer at the same time

### ISSUE THREAD

**Issue by Fictione** [artman41/guitarsniffer#17]

**Comment by Dunkalunk** [comment]
Yes unfortunately they're both pointing to the same vJoy configuration 1, I never tried setting drum sniffer to another configuration, and I'm not sure when I'll have time to try it/figure it out because it might take more than just changing the setting...

**Comment by TAOGboy** [comment]
I found a fix for it. In vjoy, add a second device with the same settings as the first device and you should be able to open both at the same time.
